README.md

```diff
@@ -4,26 +4,18 @@
 <p align="center"><b><font size="+1">
 🚧 The Slasher contract is under active development and its interface expected to change. We recommend writing slashing logic without integrating with the Slasher at this point in time. 🚧
 </font></b><p>
-EigenLayer (formerly 'EigenLayr') is a set of smart contracts deployed on Ethereum that enable restaking of assets to secure new services.
-At present, this repository contains *both* the contracts for EigenLayer *and* a set of general "middleware" contracts, designed to be reusable across different applications built on top of EigenLayer.
 
-Note that the interactions between middleware and EigenLayer are not yet "set in stone", and may change somewhat prior to the platform being fully live on mainnet; in particular, payment architecture is likely to evolve. As such, the "middleware" contracts should not be treated as definitive, but merely as a helpful reference, at least until the architecture is more settled.
+EigenLayer is a set of smart contracts deployed on Ethereum that enable restaking of assets to secure new services.
 
-Click the links in the Table of Contents below to access more specific documentation. We recommend starting with the [EigenLayer Technical Specification](docs/EigenLayer-tech-spec.md) to get a better overview before diving into any of the other docs.  For contracts addresses deployed on Goerli, click [here](https://github.com/Layr-Labs/eigenlayer-contracts/blob/master/script/output/M1_deployment_goerli_2023_3_23.json).
-## Table of Contents  
-* [Introduction](#introduction)
-* [Installation and Running Tests / Analyzers](#installation)
-* [EigenLayer Technical Specification](docs/EigenLayer-tech-spec.md)
+We recommend starting with the [technical documentation](docs/README.md) to get an overview of the contracts before diving into the code. 
+
+For deployment addresses on both mainnet and Goerli, see [Deployments](#deployments) below.
 
-Design Docs
-* [Withdrawals Design Doc](docs/Guaranteed-stake-updates.md)
-* [EigenPods Design Doc](docs/EigenPods.md)
+## Table of Contents  
 
-Flow Docs
-* [EigenLayer Withdrawal Flow](docs/EigenLayer-withdrawal-flow.md)
-* [EigenLayer Deposit Flow](docs/EigenLayer-deposit-flow.md)
-* [EigenLayer Delegation Flow](docs/EigenLayer-delegation-flow.md)
-* [Middleware Registration Flow for Operators](docs/Middleware-registration-operator-flow.md)
```

Comment:
why are we getting rid of all of these? Or we just let them navigate the docs/ folder by themselves?

Reply:
None of those links function anymore. 

The README file in the docs folder serves as a good table of contents into the rest of the docs, but the specific files I'm removing here are really out of date. I think the material should be covered by the new docs - do you see gaps anywhere?